include/quda.h

```diff
@@ -802,14 +802,25 @@ extern "C" {
 
   typedef struct QudaGaugeObservableParam_s {
     size_t struct_size; /**< Size of this struct in bytes.  Used to ensure that the host application and QUDA see the same struct*/
-    QudaBoolean su_project;              /**< Whether to project onto the manifold prior to measurement */
-    QudaBoolean compute_plaquette;       /**< Whether to compute the plaquette */
-    double plaquette[3];                 /**< Total, spatial and temporal field energies, respectively */
-    QudaBoolean compute_qcharge;         /**< Whether to compute the topological charge and field energy */
-    double qcharge;                      /**< Computed topological charge */
-    double energy[3];                    /**< Total, spatial and temporal field energies, respectively */
-    QudaBoolean compute_qcharge_density; /**< Whether to compute the topological charge density */
+    QudaBoolean su_project;               /**< Whether to project onto the manifold prior to measurement */
+    QudaBoolean compute_plaquette;        /**< Whether to compute the plaquette */
+    double plaquette[3];                  /**< Total, spatial and temporal field energies, respectively */
+    QudaBoolean compute_polyakov_loop;    /**< Whether to compute the temporal Polyakov loop */
+    double ploop[2];                      /**< Real and imaginary part of temporal Polyakov loop */
+    QudaBoolean compute_gauge_loop_trace; /**< Whether to compute gauge loop traces */
+    double_complex *traces;               /**< Individual complex traces of each loop */
+    int **input_path_buff;                /**< Array of paths */
+    int *path_length;                     /**< Length of each path */
+    double *loop_coeff;                   /**< Multiplicative factor for each loop */
+    int num_paths;                        /**< Total number of paths */
+    int max_length;                       /**< Maximum length of any path */
+    double factor;                        /**< Global multiplicative factor to apply to each loop trace */
+    QudaBoolean compute_qcharge;          /**< Whether to compute the topological charge and field energy */
+    double qcharge;                       /**< Computed topological charge */
+    double energy[3];                     /**< Total, spatial and temporal field energies, respectively */
+    QudaBoolean compute_qcharge_density;  /**< Whether to compute the topological charge density */
     void *qcharge_density; /**< Pointer to host array of length volume where the q-charge density will be copied */
+    QudaBoolean remove_staggered_phase; /**< Whether or not the resident gauge field has staggered phases applied and if they should be removed */
```

Comment:
only reading the doxygen leacves me confused. The description mentions phase applied true/false and remove_phase true/false. But there is only a single bool here. Please clarify.

Reply:
tl;dr this is a bit of a MILC-specific kludge to deal with this issue: https://github.com/lattice/quda/issues/1315 . I can add a direct reference to that issue in a comment.

Reply:
https://github.com/lattice/quda/pull/1308/commits/ccb7dd9dfe1dbb485f0d2f05456c9276a79fbafb